WordPress/theme-check
# Try: Add simplified readme validator

Fixes https://github.com/WordPress/theme-check/issues/375

This check and the accompanying parser is copied from the plugin readme validator.

It is more limited than the plugin readme validator, and it does not work for themes that are in sub folders.
The subfolder issue may be solved by #370. Untested.

**How to test:**

1. Test with a theme that is missing a readme.txt and readme.md
2. Confirm that the error about the missing readme is displaying.
3. Test with a theme with missing headers in the readme.
4. Confirm that the warnings are displaying.
5. Test with a theme with incorrect headers in the readme.
6. Confirm that the warnings are displaying.
